website/content/docs/upgrading/upgrade-specific.mdx

```diff
@@ -515,6 +515,11 @@ to Consul 1.11.11 or later to avoid the breaking nature of that change.
 
 ### Licensing Changes <EnterpriseAlert inline />
 
+You can only upgrade to Consul Enterprise 1.10 from a version of
+Consul Enterprise 1.8 >= 1.8.13 or 1.9 >= 1.9.7. Other versions of
```

Comment:
is this more clear? wasn't sure if it was 1.8.13+ or has to be between 1.8 and 1.8.13

```suggestion
Consul Enterprise 1.8.13+ or 1.9.7+. Other versions of
```

Reply:
I agree that the current wording isn't clear (copied it from another page).

I'm concerned that the proposed change might imply that 1.9.5 is acceptable, when it really isn't. 1.9.5 is technically "1.8.13+" ... but it won't work. If you're on the 1.9 series, you need 1.9.7+. If you're on the 1.8 series, you need 1.8.13+.

Reply:
Changing to:

You can only upgrade to Consul Enterprise 1.10 from the following Enterprise versions:
- 1.8 release series: 1.8.13+
- 1.9 release series: 1.9.7+